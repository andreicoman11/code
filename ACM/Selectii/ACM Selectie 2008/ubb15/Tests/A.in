2 1
2 2
3 1
3 2
3 3
17 1000000
1 1
1 2
17 35000000
17 30000000
17 1000000
17 100000
17 50000
17 19999
17 12345
18 12345
16 12345
19 12345
20 1767263189
20 1767263190
20 1767263191
20 1
20 142
20 1111111111
0 0
