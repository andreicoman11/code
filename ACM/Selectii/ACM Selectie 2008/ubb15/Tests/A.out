ES
ERROR
EESS
ESES
ERROR
EEEEEEEEEESSSESSSSSEEESSSSESESSS

ERROR
ESESESEEESEESSESSSESEESEEESSSSES
ESEESEESSEEESESEESSSESSSEESESSES
EEEEEEEEEESSSESSSSSEEESSSSESESSS
EEEEEEESEESSESESSSEESSEESSESSSSS
EEEEEESEEEEESSSSESSEESEESSSSSSSS
EEEEEEEESSEESESEESSSESESSESSSSSS
EEEEEEESEEESESEESSEEESSSSSSSSSSS
EEEEEEEESEEESESEESSEEESSSSSSSSSSSS
EEEEEESEEESESEESSEEESSSSSSSSSS
EEEEEEEEESEEESESEESSEEESSSSSSSSSSSSS
ESESESESESESESESESESESESESESESESESEESS
ESESESESESESESESESESESESESESESESESESES
ERROR
EEEEEEEEEEEEEEEEEEESSSSSSSSSSSSSSSSSSS
EEEEEEEEEEEEEEESEEESSSSSESSSSSSSSSSSSS
EEESSSEEESSSEESESESEEEESSESSSSEEESSSES
